en/Data Structures/Arrays/array.md

```diff
@@ -0,0 +1,27 @@
+# Arrays
+
+Arrays are the most basic data structures used in programming. Arrays are data structures that contain a group of elements. Typically these elements are all of the same data type, such as `char` or `int`.\
+All arrays consist of contiguous memory locations. We can access the elements with their index. The lowest address corresponds to the first element, and the highest address to the last element.
+
+<p align="center">
+    <img width="60%" src="https://www.tutorialspoint.com/cprogramming/images/arrays.jpg">
+</p>
+
+**Array Declaration**
```

Comment:
The declaration, initialization works for only certain c-like languages. Would consider removing them.

Reply:
Also do this for the Array initialization and element access. They are not important nor unique to arrays.

Reply:
Done👍🏻